### PR TITLE
Add code from previous project

### PR DESCRIPTION
**Pull request by Profsain**
- Added index.html file with HTML copied code
- Added main.css file with CSS copied code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
```
Comment (Profsain) [inline index.html:11]: Please use a semantic header tag in your title and also change your attribute from id to class, this will make your CSS code reusable.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <h3 id="img-caption">Dr Norman in the field</h3>
+      </div>
+      <p id="tribute-info">Here's a time line of Dr. Borlaug's life:</p>
+      <a
+        href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+        id="tribute-link"
+        target="_blank"
+        >Read More.....</a
+      >
+    </div>
```
Comment (Profsain) [inline index.html:27]: Please use HTML 5 main tags for your web page main section.  This will help in search engine indexing and make your code more readable.
Also, replace the id attribute with the class attribute to make your CSS code reusable.

**Review by Profsain — COMMENTED** [review]
# Changes below are required.
Kindly setup linters for HTML & CSS.  Check for any linters error by clicking on the Detail link next to the error.
Add professional describe your project in the README.md file

File: index.html (hunk 1 above)
Comment (danmainah) [reply index.html:11]: - [ ] Great review @Profsain, however, your comment is a little bit confusing, you could, for instance, rephrase it to a comment like this `Please use a semantic header tag in your title also change your attribute from id to class, this will make your CSS code reusable.`

File: index.html (hunk 2 above)
Comment (danmainah) [reply index.html:27]: - [ ] Kindly indicate why the developer needs to use the `<main>` tag.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Profsain , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good use of polite language.

# Required Changes ♻️



_Check the comments under the review._

## Optional suggestions
- [ ] Kindly ensure you begin your review by appreciating the developer's effort and progress before requesting changes.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by phelian23 — APPROVED** [review]
Hi @Profsain, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
